bidict/_defaultbidict.py

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8 -*-
+# Copyright 2009-2020 Joshua Bronson. All Rights Reserved.
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+"""Provides :class:`defaultbidict`, bidirectional defaultdict"""
+
+from collections.abc import Hashable
+from ._mut import MutableBidict
+from ._delegating import _DelegatingMixin
+
+
+class defaultbidict(_DelegatingMixin, MutableBidict):  # noqa: N801; pylint: disable=invalid-name
+    """Bidirectional defaultdict"""
+    __slots__ = ('_default_factory', )
+
+    def __init__(self, *args, **kw):
+        """Bidirectional defaultdict.
+        Provide :attr:`default` to set default factory"""
+        default_factory = kw.pop('default', None)
+        super().__init__(*args, **kw)
+        if default_factory is not None and not (default_factory, Hashable):
```

Comment:
The tuple `(default_factory, Hashable)` will always be truthy. Looks like you meant `isinstance(default_factory(), Hashable)`.